purchases_ui_flutter/ios/Classes/PurchasesUiPaywallView.swift

```diff
@@ -57,6 +57,10 @@ class PurchasesUiPaywallView: NSObject, FlutterPlatformView {
             if let offeringId = args["offeringIdentifier"] as? String {
                 _paywallViewController.update(with: offeringId)
             }
+            if let displayCloseButton = args["displayCloseButton"] as? Bool {
+                // TODO: Set displayCloseButton
+                // TODO: set onDismiss callback
```

Comment:
I'm holding this since in iOS, we will need https://github.com/RevenueCat/purchases-ios/pull/3738